.env.example

```diff
@@ -45,12 +45,33 @@ KAFKA_TRANSACTION_STATE_LOG_MIN_ISR=1
 KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR=1
 
 # ============================================
-# Frontend Configuration
+# Service Configuration
 # ============================================
-# Used by: frontend (both compose files)
+# Docker container ports
 FRONTEND_PORT=5173
 BACKEND_PORT=8000
```

Comment:
I in general go for custom port numbers everywhere. That way if someone has other projects running on the machine they won't overlap. At least main frontend and main backend ports. Internal services like redis/db don't need to be exposed outside the docker compose network, so they can stay default.